### PR TITLE
sites compatibility

### PR DESCRIPTION
**Pull request by reepush**
– **wasabisys.com** compatibility
– **ims.brk.de** compatibility
– **chase.com** compatibility
– **pwc.com** compatibility
– **sky.it** compatibility
– ignore mailformed action for form
– don't autosubmit form with defined credentials
– fix custom credentials issue

### CODE REVIEW

**Comment by reepush** [comment]
@limpkin will write ci results soon

**Comment by limpkin** [comment]
@mosaadm will fix CI so we can merge it :)

File: chrome_extension/vendor/mooltipass/mcCombinations.js [hunk 1]
```diff
@@ -1422,7 +1422,7 @@ mcCombinations.prototype.retrieveCredentialsCallback = function (credentials) {
 			if (a.deep == b.deep) return 0
 		})
 		
-		if (buttons.length > 0) return buttons[0]
+		if (buttons.length > 0 && buttons[0].deep < 150) return buttons[0]
```
Comment (limpkin) [inline chrome_extension/vendor/mooltipass/mcCombinations.js]: are you sure that's a good idea? what happens if that fails? no submit event?

Comment (reepush) [reply chrome_extension/vendor/mooltipass/mcCombinations.js]: @limpkin we will fallback to form submit event if it fails

Comment (reepush) [reply chrome_extension/vendor/mooltipass/mcCombinations.js]: i think it's better than triggering click event on the wrong element, users report this kind of issue quite often

Comment (limpkin) [reply chrome_extension/vendor/mooltipass/mcCombinations.js]: good point

**Comment by reepush** [comment]
[master](https://travis-ci.org/reepush/mooltipass/builds/319360132) and [sites-compatibility](https://travis-ci.org/reepush/mooltipass/builds/319351383)

@limpkin I checked anibis.ch and kenwoodworld.com manually and it works (issue with first one is caused by captcha, not sure what's wrong with second though)

**Comment by limpkin** [comment]
@reepush not that I want to spend some time comparing, but i'm guessing we're good to go?

**Comment by reepush** [comment]
@limpkin yeah, I noticed these failures because I'm not sure what caused them

**Pull request merged**